packages/nextjs/src/components/_common/headers/Header.tsx

```diff
@@ -18,18 +18,15 @@ export const Header = ({ children, size, level, anchorId, className }: Props) =>
     const anchor = anchorId ? (anchorId.startsWith('#') ? anchorId : `#${anchorId}`) : undefined;
     const fallbackSizeByLevel = levelToSize[level] || 'large';
 
-    return (
-        <div className={classNames(style.header, className)}>
-            <div className={style.anchorOffset} id={anchorId} tabIndex={-1} />
-            <Heading size={size || fallbackSizeByLevel} level={level}>
-                {anchor && (level === '2' || level === '3') ? (
-                    <a href={anchor} className={style.anchorLink}>
-                        {children}
-                    </a>
-                ) : (
-                    children
-                )}
-            </Heading>
-        </div>
+    return (        
+        <Heading id={anchorId} size={size || fallbackSizeByLevel} level={level} className={classNames(style.header, className)}>
+            {anchor && (level === '2' || level === '3') ? (
```

Comment:
Code smell analyzeren vil heller at vi bruker nullish coalescing istedenfor or``
```suggestion
            {anchor && (level === '2' ?? level === '3') ? (
```

Reply:
Det er utenfor hva jeg har kunnskap om, så her må noen andre bestemme.

Reply:
Jeg tror at || er det vi faktisk vil bruke i dette tilfellet. Jeg markerer den som falsk positiv i SonarCloud jeg om ingen protesterer